content/en/monitors/types/process.md

```diff
@@ -43,11 +66,17 @@ After defining your search, a graph is displayed above the search inputs with an
 
 ### Set alert conditions
 
-* The count was `above`, `above or equal to`, `below`, or `below or equal to`
-* the threshold during the last `5 minutes`, `15 minutes`, `1 hour`, etc. or `custom` to set a value between 1 minute and 48 hours.
+- The count was `above`, `above or equal to`, `below`, or `below or equal to`
+- the threshold during the last `5 minutes`, `15 minutes`, `1 hour`, etc. Additionally, you can use `custom` to set a value between 5 minutes and 24 hours.
```

Comment:
Not quite sure what to do about the `vale` warning here. I think "`5 minutes`, `15 minutes`, `1 hour`, and more" sounds a bit strange.

Reply:
What about "or larger time frames"?

Reply:
That sounds good to me!